BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -83,25 +85,30 @@ private void updateLocation() {
             Collections.sort(usableBeacons, Beacon.RssiComparator);
             Collections.reverse(usableBeacons);
```

Comment:
Instead of sorting, reversing, iterating over the beacons and throwing some away, we could just sort, start at index 3 and break once we find one which doesn't match or if the index gets too high and delete all at once.